packages/grafana-ui/src/types/datasource.ts

```diff
@@ -1,9 +1,9 @@
-import { ComponentType, ComponentClass } from 'react';
-import { TimeRange } from './time';
-import { PluginMeta, GrafanaPlugin } from './plugin';
-import { TableData, TimeSeries, SeriesData, LoadingState } from './data';
-import { PanelData } from './panel';
+import { ComponentClass, ComponentType } from 'react';
```

Comment:
not sure why my vscode rearranges the imports... is this desired?

Reply:
Does not happen for others, maybe a setting that is on for you? or some plugin?